### DISCUSSION TITLE
UI: Only stacked graphs should use fill by default

### DISCUSSION THREAD

**Discussion by RichiH** [grafana/grafana#36967] [Dashboards, Panels & Visualizations]
It's quite confusing to have non-stacked graphs using line fill by default.

The only corner case is having one single graph in which you could arguably use fill to show that this is all there is to see.

To put this another way: The default config should ensure that any user can instantly see if a certain panel uses stacked or non-stacked graphs.

**Comment by pkolyvas** [comment]
The default Time series panel in 8+ no longer has a fill style by default. I shouldn't have migrated this issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
